README.md

```diff
@@ -22,15 +22,17 @@ The main objective is to showcase that a user can take a trained model, use a pi
 1. Register the clusters
 
    [ACM Application](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications) manifests are located in [acm/registration](acm/registration) to register and configure the target environments required for the AI at the Edge use cases.  The files can be applied to the ACM hub cluster manually:
+
+   ```bash
+   oc apply -k acm/registration
    ```
-   $  oc apply -k acm/registration
-   ```
+
    * Core - Cluster host the ODH Core components that will be used in the MLOps Engineer workflow to train, build and push the model.  This cluster is not required to be co-located with the ACM Hub but we group them together to simplify the use case
    * Near Edge - Cluster(s) that will host the running model at the edge.  This is the target environment after a new model is available for use
-1. Deploy Open Data Hub to the Core cluster and register any configurations to support pushing models to the edge cluster
+2. Deploy Open Data Hub to the Core cluster and register any configurations to support pushing models to the edge cluster
    * GitOps repos
    * Image repos
-1. Manage the edge cluster environments to support deployment of models and support for monitoring
+3. Manage the edge cluster environments to support deployment of models and support for monitoring
```

Comment:
I'd go back to the auto-generated numbers here, since they're used for the rest of the numbers in the list. Even though you're changing to 2 and 3 here, GitHub rendering seems to be fixing it back to 5 and 6 as expected.

If your local rendering engine is not replacing the numbers correctly, it might be because of the blank lines introduced here on lines 25 and 29? I'm not sure though

Reply:
Must've done that out of habit, my bad should be fixed now.